octue/conventional-commits
# Add `check-semantic-version` GitHub action

# Summary
Add a GitHub action utilising the `check-semantic-version` CLI that can be easily used in GitHub workflows.


# Contents ([#76](https://github.com/octue/conventional-commits/pull/76))

### New features
- Add `check-semantic-version` GitHub action

### Operations
- Use latest `setup-python` action in workflows
- Use `check-semantic-version` GitHub action in `python-ci` workflow
- Add `build-docker-image` workflow
- Add testing and docker build/push to `release` workflow

### Other
- Update example workflow with `check-semantic-version` GHA
